### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by paul0vinicius**
#### What is the purpose of this pull request?

Update its README.

#### What problem is this solving?

The lackness of documentation.

#### Types of changes
- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,74 +18,113 @@ See our [LTS policy](https://github.com/vtex-apps/awesome-io#lts-policy) for mor
...
-| `allowSwap`        | `Boolean`         | Allow or not the item swap                    |
-| `allowRemoval`     | `Boolean`         | Allow or not the item removal                 |
+Through the Storefront you can change the behavior and interface of `ProductKit`. But, you can also make adjusts in your theme app, like Store does.
+
+| Prop name          | Type              | Description                                   | Default Value |
+| ------------------ | ----------------- | --------------------------------------------- |-------------- |
+| `showArrows`       | `Boolean`         | Show or not the arrows                        | true |
+| `nextArrow`        | `String`          | Next arrow icon                               | `''` |
+| `prevArrow`        | `String`          | Previous arrow icon                           | `''` |
+| `showDots`         | `Boolean`          | Show or not the dots                          | true |
```
Comment (theomoura) [inline README.md]:
```suggestion
| `showDots`         | `Boolean`          | Show or not the slider dots                          | true |
```

File: README.md [hunk 2]
```diff
@@ -18,74 +18,113 @@ See our [LTS policy](https://github.com/vtex-apps/awesome-io#lts-policy) for mor
...
+| `dots`             | `String`          | Dots icon                                     | `''` |
```
Comment (theomoura) [inline README.md]:
```suggestion
| `dots`             | `String`          | Slider dots icon                                     | `''` |
```

File: README.md [hunk 3]
```diff
@@ -18,74 +18,113 @@ See our [LTS policy](https://github.com/vtex-apps/awesome-io#lts-policy) for mor
...
+| `showListPrice`    | `Boolean`         | Show or not the list price                    | true |
+| `showLabels`        | `Boolean`         | Show or not the labels "from" and "to"        | false |
+| `showInstallments` | `Boolean`         | Show or not the installments                  | false |
+| `showBadge`        | `Boolean`         | Show or not the discount badge                | false |
+| `badgeText`        | `String`          | Text of the discount badge                    | `Texto da badge/Badge Text/Texto de badge` |
+| `showCollections`  | `Boolean`         | Show or not the collections badges            | true |
+| `allowSwap`        | `Boolean`         | Allow or not the item swap                    | true |
+| `allowRemoval`     | `Boolean`         | Allow or not the item removal                 | true |
+
+Also, you can configure the product summary that is defined on minicart. See [here](https://github.com/vtex-apps/product-summary/blob/master/README.md#configuration) the Product Summary API.
```
Comment (GustavoMA) [inline README.md]:
```suggestion
Also, you can configure the product summary that is defined on product kit. See [here](https://github.com/vtex-apps/product-summary/blob/master/README.md#configuration) the Product Summary API. 
```

File: README.md (hunk 1 above)
Comment (paul0vinicius) [reply README.md]: nice touch

**Pull request merged**